.circleci/config.yml

```diff
@@ -342,7 +342,7 @@ jobs:
             kubectl cluster-info dump > /go/out/logs/cluster-info.dump.txt
             kubectl describe pods -n pilot-auth > /go/out/logs/pods-system.txt
             kubectl describe pods -n pilot-auth-system > /go/out/logs/pods-test.txt
-            /go/bin/go-junit-report </go/out/tests/test-report-auth-pilot-v1.raw > /go/out/tests/test-report-auth-pilot-v1.xml
+            cat /go/out/tests/*.raw | /go/bin/go-junit-report > /go/out/tests/test-report-noauth-pilot.xml
```

Comment:
isn't this running auth tests, `make test/minikube/auth/e2e_pilot_alpha1 HUB="${HUB}" TAG="${TAG}"` above?

Reply:
Separate PR (given the long wait for a PR to submit)